.circleci/config.yml

```diff
@@ -66,25 +66,19 @@ commands:
       - restore_cache:
           keys:
             # when lock file changes, use increasingly general patterns to restore cache
-            - gears-{{ .Branch }}-{{ checksum "tests/flow/get_gears.sh" }}
-            - gears-{{ .Branch }}
-            - gears-
-      - run:
-          name: build gears
-          no_output_timeout: 30m
-          command: |
-            cd tests/flow
-            ./get_gears.sh
+            - gears-{{ arch }}-{{ .Branch }}-{{ checksum "sbin/getgears" }}
+            - gears-{{ arch }}-{{ .Branch }}
+            - gears-{{ arch }}-
       - save_cache:
           paths:
-            - /tmp/gears
-          key: gears-{{ .Branch }}-{{ checksum "tests/flow/get_gears.sh" }}
+            - ./bin/linux-buster-x64/RedisGears
```

Comment:
@rafie you need to call get gears before you save it to cache.

Reply:
@danni-m The `sbin/getgears` script will download the proper Gears from S3 instead of cloning and building (which only take fraction of the time). I'd completely avoid caching of those. I'm pretty afraid of the cache, anyway. Can cause strange issues.